fontc/src/workload.rs

```diff
@@ -191,7 +196,15 @@ impl<'a> Workload<'a> {
         // Async work will send us it's ID on completion
         let (send, recv) = crossbeam_channel::unbounded::<(AnyWorkId, Result<(), AnyWorkError>)>();
 
-        rayon::in_place_scope(|scope| {
+        // a flag we set if we panic
+        let abort_queued_jobs = Arc::new(AtomicBool::new(false));
+        // build a custom threadpool. we use this to ensure all threads are named
+        let threadpool = ThreadPoolBuilder::new()
+            .thread_name(|n| format!("tp{n}"))
```

Comment:
Unless we can assign useful names I wonder how much better than thread ids this is? Not worse I suppose.

Reply:
I was finding it annoying that the `ThreadId` type didn't impl Display, and the Debug impl was longer than necessary, which made our log preambles longer. With this, we will print `tp3` instead of `ThreadId(3)`. 🤷